recipes/libsimpleitk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: libsimpleitk
+  version: "{{version}}"
+
+source:
+  git_url: git://itk.org/SimpleITK.git
+  git_tag:  v{{version}}
+  patches:
+    - fixes.patch
+
+build:
+    number: 0
+    skip: True   # [win] missing lua dependecy
+    features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py>=35]
+
+requirements:
+  build:
+    - cmake    >=3.3
+    - ninja
+    - lua      5.1.*
+    - swig     >=3
+    - virtualenv
+    - itk     # 4.11.0
+    - python  # [win]
+    - vc 9    # [win and py27]
+    - vc 10   # [win and py34]
+    - vc 14   # [win and py35>=35]
+  run:
+    - itk    # 4.11.0
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35>=35]
+
+test:
+  commands:
+    - echo "Testing..."
```

Comment:
Are there some file tests we could do? Like checking for headers and libraries that are expected?